Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.Handtracking.Editor.asmdef

```diff
@@ -1,8 +1,9 @@
 {
-    "name": "Microsoft.MixedReality.Toolkit.XRSDK.Oculus.Handtracking.Editor",
+    "name": "Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.Handtracking.Editor",
```

Comment:
We should keep an eye on this file's name with regards to https://github.com/microsoft/MixedRealityToolkit-Unity/issues/8511. It's pretty long!